modules/heat_conduction/test/tests/gap_heat_transfer_mortar/gold/closed_gap_prescribed_pressure_out.csv

```diff
@@ -0,0 +1,3 @@
+time,aluminum_pt_interface_temperature,interface_heat_flux_aluminum,interface_heat_flux_steel,steel_pt_interface_temperature
+0,0,0,0,0
+1,283.85580189521,-7292.0186846471,7292.0186662386,349.87541761867
```

Comment:
Out of curiosity, do you have analytical data on this?

Reply:
Never mind, I do see in your input files you do have an analytical reference.

Reply:
Is there a better place for the analytical solution and calculation? I wasn't quite sure where to put that information (but since I worked it all out, I selfishly wanted to put it somewhere)

Reply:
Input file works for me, especially if there isn't a reference available that you can't use in the docs.

Reply:
I just missed this initially since I only looked at your first commit 😅